dataset/src/test/scala/frameless/ExplodeTests.scala

```diff
@@ -48,4 +48,19 @@ class ExplodeTests extends TypedDatasetSuite {
     check(forAll(prop[Int] _))
     check(forAll(prop[String] _))
   }
+
+  test("explode on Maps") {
+    def prop[A: TypedEncoder: ClassTag](xs: List[X1[Map[A, A]]]): Prop = {
+      val tds = TypedDataset.create(xs)
+
+      val framelessResults = tds.explodeMap('a).collect().run().toVector
```

Comment:
@OlivierBlanvillain @imarios I would need you help here if possible since I know nothing about shapeless :/
The compiler is complaining about 
`No column Symbol with shapeless.tag.Tagged[String("a")] of type scala.collection.immutable.Map[A,B] in frameless.X1[scala.collection.immutable.Map[A,A]]`

Am I missing something in the definition of `explodeMap` ?

Reply:
Hey @ayoub-benali, sorry for the delay, I was trying to get the new version out and then help another older PR to get merged. Let me take a look and help you out